src/components/product-grid/product-grid.module.scss

```diff
@@ -19,6 +19,6 @@
 
 @media (max-width: 1024px) {
     .productGrid {
-        --minCardSize: 250px;
+        grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
```

Comment:
I prefer to have a single calculation and change a variable, but maybe this works better for Codux UX

Reply:
Changed to a pair of variables for min and max:
```css
/* The max width must be at least 3/2 of the min width to prevent gaps when
 * transitioning from 3 to 2 items per row. */
--minColumnWidth: 350px;
--maxColumnWidth: 540px;
```